build.sbt

```diff
@@ -19,7 +19,7 @@ lazy val root = (project
   settings commonSettings
   settings (
     publish / skip := true,
-    name := "smetrics")
+    name := "smetrics-parent")
```

Comment:
why do we need this?

Reply:
because otherwise there are two sbt modules named `smetrics` and it wrecks IDEA's project structure